libs/wingc/src/type_check/jsii_importer.rs

```diff
@@ -102,6 +103,8 @@ impl<'a> JsiiImporter<'a> {
 					self.wing_types.json()
 				} else if type_fqn == &format!("{}.{}", WINGSDK_ASSEMBLY_NAME, WINGSDK_MUT_JSON) {
 					self.wing_types.mut_json()
+				} else if type_fqn == &format!("{}.{}", WINGSDK_ASSEMBLY_NAME, WINGSDK_STRUCT) {
+					self.wing_types.anything()
```

Comment:
Does this actually happen? Can leave it out if not

Reply:
I thought it did but it appears it does not, removed 👍